simpler-wick.sty

```diff
@@ -193,7 +193,7 @@
   \tikz[remember picture, overlay]
   % L. added contraction width option, also adjusted the start of the contraction line to be a bit lower, 
   % while initially fine, it looked a bit off for the bottom contraction so I altered this to be consistent
```

Comment:
Similarly, this comment is not needed in the final code (fine in a pull request, but it doesn't need to stay in the long run)